### PR TITLE
Refactor bootstrap command generation

### PR DESCRIPTION
**Pull request by hebbo**
## Approvals

- [x] Verified
- [ ] @TimMoore
- [x] @hebbo
- [x] @ptolemy428
- [x] @iiman

- Move get_env_vars_list into new launch_configuration module
- Create into new launch_configuration module into new launch_configuration module
- Create test for into new launch_configuration module

The code within the following commit make use of the changes:
https://github.com/cultureamp/cloudformation-stacks/pull/93/commits/d57715d2249b67410080fd3d90b6beb280d2afbf

### CODE REVIEW

**Comment by hebbo** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;KRHzzMEZ&#x2F;523-add-get-env-vars-list-funciton-to-cfn-core"> Add get_env_vars_list funciton to cfn-core</a>

File: tests/unit/test_launch_configuration.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    def test_should_concat_env_var_list(self):
+        expected = ["ENVVAR1='value1'", "ENVVAR2='value2'"]
+        input = {"envvar1": "value1", "envvar2": "value2"}
+
+        actual = get_env_vars_list(input)
+        self.assertItemsEqual(expected, actual,
+                              "env var list concatenated incorrectly")
+
+    def test_should_create_bootstrap_command(self):
+        expected = "ENVVAR2='value2' ENVVAR1='value1'" + " " + "ansible-playbook /opt/cultureamp/bootstrap.yml"
```
Comment (TimMoore) [inline tests/unit/test_launch_configuration.py]: Why not specify it all as one string... there's no point concatenating three constants.

**Comment by iiman** [comment]
🏁

File: tests/unit/test_launch_configuration.py (hunk 1 above)
Comment (hebbo) [inline tests/unit/test_launch_configuration.py]: Fixed.

**Comment by hebbo** [comment]
Merging in 3... 2... 1... :)

**Pull request merged**